client/src/utils/attributeAndParamUtils.js

```diff
@@ -501,6 +501,25 @@ const initSsotSessionStorage = (robotId) => {
         console.error(error);
       });
 };
+
+/**
+ * @description Will send a backend call to delete a robot
+ * @param {String} robotId Id of the robot that is deleted
+ */
+const deleteRobotFromDB = async (robotId)  => {
+  const requestStringParameters = `/ssot/delete/${robotId}`;
+  const deleteMethod = {
```

Comment:
I'd prefer this inline or with a better naming, took me a couple of minutes to figure out what you are doing with this